fe/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
I know it might be react thing or boilerplate default, but personally I find it weird and wrong to have css classes and ids have capitalizations. This is up to larger audience to decide.

Reply:
Hmm, yep this looks not that good, I won't bother changing it as we'll switch to scss modules later on anyway

Reply:
yeah I thought that as well, but had to gripe ;P just to see if I was only one 👍